lib/ClangImporter/ClangIncludePaths.cpp

```diff
@@ -540,6 +550,12 @@ ClangInvocationFileMapping swift::getClangInvocationFileMapping(
                                          StringRef("SwiftGlibc.h"), vfs);
   }
   result.redirectedFiles.append(libcFileMapping);
+  if (triple.isAndroid()) {
+    // Android uses the android-specific module map that overlays the NDK.
+    // FIXME: Drop Glibc mapping for android as well.
+    result.redirectedFiles.append(
+        getLibcFileMapping(ctx, "android.modulemap", std::nullopt, vfs));
```

Comment:
I think `bionic.modulemap` would be best, following after the naming of glibc and musl.

Reply:
I do agree with you in some aspect, I would prefer `bionic.modulemap` however given that it is covering zlib as well - I think that `android.modulemap` also works - it basically covers the NDK rather than just bionic.

Reply:
I would prefer to stick with `android.modulemap` too